v22.2/trigram-indexes.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Trigram Indexes
+summary: Trigram indexes on STRING columns improve the performance of fuzzy string matching and pattern matching in large datasets.
+toc: true
+keywords: gin, gin index, gin indexes, inverted index, inverted indexes, accelerated index, accelerated indexes, trigram, trigrams
+docs_area: develop
+---
+
+A _trigram index_ is a type of [inverted index](inverted-indexes.html) created on a [`STRING`](string.html) column. Trigram indexes are used to efficiently search for strings in large tables without providing an exact search term.
```

Comment:
Is this new to v22.2? Maybe add the "New in" tag to the start of this paragraph?